src/util/buttonActions.js

```diff
@@ -0,0 +1,9 @@
+export const disabledButton = (name) => {
```

Comment:
동사형으로 작성하는 게 더 자연스러워 보입니다!
```suggestion
export const disableButton = (name) => {
```